src/js/components/Drop/stories/CustomThemed/Lazy.js

```diff
@@ -22,6 +22,23 @@ const LazyDrop = () => {
   const bottomLeftTargetRef = useRef();
   const bottomRightTargetRef = useRef();
 
+  const alignBottomLeft = useMemo(() => {
```

Comment:
Eric mentioned this in the site example review, but since this is really a constant that doesn't depend on anything else it should outside the component function. It doesn't need memoizing really. It just needs to be actually constant.